guides/release/index.md

```diff
@@ -1,117 +1,28 @@
-making costly architectural mistakes that are much harder to fix if you only
-notice them once your app is nearly complete.
-
-In short:
-
-1. Always test on real, representative mobile devices.
-2. Measure performance from the beginning, and keep testing as your app
-   develops.
-
-These tips will help you identify problems early so they can be addressed systematically, rather than
-in a last-minute scramble.
-
-## Reporting a problem
-
-Typos, missing words, and code samples with errors are all considered
-documentation bugs. If you spot one of them, or want to otherwise improve
-the existing guides, we are happy to help you help us!
-
-Some of the more common ways to report a problem with the guides are:
-
-* Using the pencil icon on the top-right of each guide page
-* Opening an issue or pull request to [the GitHub repository][gh-guides]
-
-Clicking the pencil icon will bring you to GitHub's editor for that
-guide so you can edit right away, using the Markdown markup language.
-This is the fastest way to correct a typo, a missing word, or an error in
-a code sample.
-
-If you wish to make a more significant contribution be sure to check our
-[issue tracker][gh-guides-issues] to see if your issue is already being
-addressed. If you don't find an active issue, open a new one.
-
-If you have any questions about styling or the contributing process, you
-can check out our [contributing guide][gh-guides-contributing]. If your
-question persists, reach us in the `#dev-ember-learning` channel on the [Ember Community Discord][discord].
-
-Good luck!
-
-[ember-cli]: https://ember-cli.com/
-
-[mdn]: https://developer.mozilla.org/en-US/docs/Web
-[promises]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
-[js-modules]: http://jsmodules.io/
-[babeljs]: https://babeljs.io/docs/learn-es2015/
-[es6]: https://leanpub.com/understandinges6/read
-
-[gh-guides]: https://github.com/ember-learn/guides-source/
-[gh-guides-issues]: https://github.com/ember-learn/guides-source/issues
-[gh-guides-contributing]: https://github.com/ember-learn/guides-source/blob/master/CONTRIBUTING.md
-
-[discord]: https://discordapp.com/invite/zT3asNS
+Welcome to the Ember.js Guides! This documentation will take you from total beginner to Ember expert.
+
+With the plethora of libraries readily available for front-end development, sometimes it can be a little confusing to work with a front-end framework like Ember.js, where _everything_ you need to build an application is already included. To that end, we've segmented out each part of the guides so you can focus on just the part you want to work with. This should also make it faster for you to find what you need!
+
+<ul class="list-cards">
+	<li class="list-item-card">
+		<div class="shape shape--dark"></div>
+    <div class="shape shape--accent"></div>
+    <div class="shape shape--light"></div>
+		<a href="https://guides.emberjs.com/release/getting-started/intro/">Ember.js</a>
```

Comment:
Looks good here, will this link be accurate post merge?

Reply:
I added the intro page, so it should be. I'm not sure that's what it should be named though.

Reply:
see my other comment 👍